airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```
@@ -94,9 +94,9 @@
             }
           },
           {
-            "title": "Standard Inserts",
+            "title": "Please select another option",
             "additionalProperties": false,
-            "description": "Uses <pre>INSERT</pre> statements to send batches of records to Snowflake. Easiest (no setup) but not recommended for large production workloads due to slow speed.",
+            "description": "Please select another option",
```

removing "another" to make it a bit more natural

```suggestion
            "description": "Select an option",
```

---

```suggestion
            "description": "This mode has been deprecated. It is now equivalent to the internal staging mode, which is more performant.",
```